controllers/workspace_controller_outputs.go

```diff
@@ -174,7 +174,7 @@ func (r *WorkspaceReconciler) reconcileOutputs(ctx context.Context, w *workspace
 		}
 		if runApplied {
 			w.log.Info("Reconcile Outputs", "mgs", "run successfully applied")
-			if w.instance.Status.Run.OutputRunID != workspace.CurrentRun.ID {
+			if w.instance.Status.Run != nil && w.instance.Status.Run.OutputRunID != workspace.CurrentRun.ID {
```

Comment:
There is a different PR opened to update this logic. Making this change now to pass tests.